defaults/main.yml

```diff
@@ -1,9 +1,19 @@
 ---
-# The AWS S3 bucket containing the Nessus Debian package file
-package_bucket: "cisa-cool-third-party-production"
+# The processor architecture for which Nessus is to be installed
+architecture: amd64
 
-# The name of the Nessus Debian package file
-package_file: "Nessus-{{ version }}-debian6_amd64.deb"
+# The Linux distribution on which Nessus is to be installed
```

Comment:
I think this should be rephrased to establish that it's a Tenable configuration thang. The 10.x deb files use `debian9` and 8.x use `debian6` even though they support the exact same Linux distributions as an example.

Reply:
Please see commit f3e829d.